spec.md

```diff
@@ -1103,6 +1346,16 @@ If a `CreateVolume` operation times out, leaving the CO without an ID with which
 
 It is NOT REQUIRED for a controller plugin to implement the `LIST_VOLUMES` capability if it supports the `CREATE_DELETE_VOLUME` capability: the onus is upon the CO to take into consideration the full range of plugin capabilities before deciding how to proceed in the above scenario.
 
+##### `CreateSnapshot`, `DeleteSnapshot`, `ListSnapshots`
+
+The plugin-generated `snapshot_id` is a REQUIRED field for the `DeleteSnapshot` RPC, as opposed to the CO-generated snapshot `name` that is REQUIRED for the `CreateSnapshot` RPC: these fields MAY NOT contain the same value.
```

Comment:
s/MAY NOT/MUST NOT/ ?

Reply:
"MAY NOT" may be better because we don't have a way to ask storage systems not to use the CO-generated "snapshot_id" as the plugin's "snapshot_id".